src/App.tsx

```diff
@@ -15,48 +16,120 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NoSort,
+  Name,
+  Length,
+}
+
+enum DisplayOrder {
+  Normal,
+  Reverse,
+}
+
+type DisplayOptions = {
+  sortBy: SortType;
+  displayOrder: DisplayOrder;
+};
+
+function getPreparedGoods(
+  goods: string[],
+  { sortBy, displayOrder }: DisplayOptions,
```

Comment:
Why do you pass the second and third arguments as one object? 
What is the point in this?

Reply:
For convenience, so the order of the arguments doesn't matter.